<ui version="4.0" >
 <class>QRL_LedsManager</class>
 <widget class="QDialog" name="QRL_LedsManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Leds Manager</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QListWidget" name="ledListWidget" >
     <property name="baseSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QTabWidget" name="tabWidget" >
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab" >
        <attribute name="title" >
         <string>Led</string>
        </attribute>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QCheckBox" name="showCheckBox" >
             <property name="text" >
              <string>Show/Hide</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <item>
              <widget class="QLabel" name="minLabel" >
               <property name="text" >
                <string>Minimum:</string>
               </property>
               <property name="buddy" >
                <cstring>minLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="minLineEdit" >
               <property name="layoutDirection" >
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="text" >
                <string>-1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <item>
              <widget class="QLabel" name="maxLabel" >
               <property name="text" >
                <string>Maximum:</string>
               </property>
               <property name="buddy" >
                <cstring>maxLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="maxLineEdit" >
               <property name="layoutDirection" >
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="text" >
                <string>1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QComboBox" name="meterComboBox" />
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>207</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="helpButton" >
         <property name="text" >
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton" >
         <property name="text" >
          <string>Close Manager</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>QRL_LedsManager</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>314</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel" >
     <x>381</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
